src/ert/ensemble_evaluator/builder/_legacy.py

```diff
@@ -54,23 +57,74 @@ def __init__(  # pylint: disable=too-many-arguments
         self._job_queue = queue_config.create_job_queue()
         self._analysis_config = analysis_config
         self._config: Optional["EvaluatorServerConfig"] = None
-        self._output_bus: "asyncio.Queue[CloudEvent]" = asyncio.Queue()
+        self._output_bus: "asyncio.Queue[com_protocol.DispatcherMessage]" = (
+            asyncio.Queue()
+        )
+
+    def generate_event_creator(
+        self, experiment_id: Optional[str] = None
+    ) -> Callable[
+        [str, Optional[int]], Union[CloudEvent, com_protocol.DispatcherMessage]
+    ]:
+        if experiment_id is not None:
+
+            def func(
```

Comment:
maybe call this something more descriptive, like `node_status` ? (since we use a `node_status_builder `to make it)

Reply:
I renamed both: `node_builder` and `event_builder` respectivelly.